lib/routes/account.js

```diff
@@ -1232,106 +1228,17 @@ module.exports = function (
     {
       method: 'POST',
       path: '/account/unlock/resend_code',
-      config: {
-        validate: {
-          payload: {
-            email: validators.email().required(),
-            service: isA.string().max(16).alphanum().optional(),
-            redirectTo: validators.redirectTo(config.smtp.redirectDomain).optional(),
-            resume: isA.string().max(2048).optional()
-          }
-        }
-      },
       handler: function (request, reply) {
-        log.begin('Account.UnlockCodeResend', request)
-        var email = request.payload.email
-        var emailRecord
-        var service = request.payload.service || request.query.service
-
-        customs.check(
-          request,
-          email,
-          'accountUnlockResendCode')
-          .then(
-            db.emailRecord.bind(db, email)
-          )
-          .then(
-            function (_emailRecord) {
-              if (! _emailRecord.lockedAt) {
-                throw error.accountNotLocked(email)
-              }
-
-              emailRecord = _emailRecord
-              return db.unlockCode(emailRecord)
-            }
-          )
-          .then(
-            function (unlockCode) {
-              return mailer.sendUnlockCode(
-                emailRecord,
-                unlockCode,
-                {
-                  service: service,
-                  redirectTo: request.payload.redirectTo,
-                  resume: request.payload.resume,
-                  acceptLanguage: request.app.acceptLanguage
-                }
-              )
-            }
-          )
-          .done(
-            function () {
-              reply({})
-            },
-            reply
-          )
+        log.error({ op: 'Account.UnlockCodeResend', request: request })
```

Comment:
Wonder if it makes sense to create a follow up issue to completely remove endpoints after this rides the train for a few and we verify that no clients use this endpoint?